grails-app/controllers/org/pih/warehouse/picklist/PicklistController.groovy

```diff
@@ -94,28 +94,54 @@ class PicklistController {
         render(template: "/picklist/print", model: [requisition: requisition, picklist: picklist, location: location, order: params.order])
     }
 
+    private Map formatPicklistItemPickExport(Map item) {
+        return [
+                "${g.message(code: 'default.id.label')}": item?.id ?: "",
```

Comment:
let's add default messages

Reply:
sure, I was not thinking about it since I just copy pasted the old approach that did not have the default messages